pyNorm/io/pyn_io.py

```diff
@@ -1,3 +1,192 @@
+def save_pyn_results(spec,filename = 'None'):
+    import pickle
+    '''
+    Function to write a pickle file containing the updated pynorm calculations
+    Autogenerates a filename of the form:ion_z_redshift_ra_dec.p
+    '''
+    if filename == 'None':
+        filename = spec['ion']+'_z_'+str(spec['redshift'])+str(spec['RA'])+'_'+str(spec['Dec'])
+
+    with open(filename+'.p', "wb") as file:
+        pickle.dump(spec, file)
+
+def read_rbcodes(input_filename, targname, ra, dec, ion, partial_pixels=True):
```

Comment:
Perhaps default values could be used for RA, Dec: `ra=0.00, dec=+0.00`. That way one can run the code without assigning RA, Dec if desired.

Reply:
Check if LSR correction is done by default